docker/README.md

```diff
@@ -69,7 +69,7 @@ For example, to increase heap size, set `JAVA_OPTS` as shown in our [docker-comp
 
 For example, to add debug logging, set `command` as shown in our [docker-compose](examples/docker-compose.yml) file:
 ```yaml
-    command: --logging.level.zipkin2=DEBUG
+    command: -Dlog.level=DEBUG
```

Comment:
Can we support previous option too?

Reply:
This seems a little hard. log4j V.S. log4js. Unless we hard coded some hacking codes.